docs/src/supported.md

```diff
@@ -31,6 +31,23 @@ Additionally you can specify a set of allowed integers:
 @variable(m, x, CS.Integers([1,3,5,7]))
 ```
 
+### Anonymous variables
+
+Besides the named way of defining variables it's also possible to have anonymous variables as provided by [JuMP.jl](https://github.com/jump-dev/JuMP.jl). 
+
+This can be useful when one needs to create temporary variables for reformulations of the problem. The values of these variables can't be easily accessed later on but they avoid the problem of needing a new name for all temporary variables.
```

Comment:
Perhaps also note that if the anonymous variables are returned from a function (constraint) then they can be used as a variables in the main model, e.g. for printing/debugging.

Reply:
Do you mind having another look?

Reply:
Yes, Great with a complete (although a bit contrived) model. I say go!

Reply:
Haha yeah was the simplest even though kinda useless model I could think off. 
It can always be improved :smile: